server/data/apps.json

```diff
@@ -3840,6 +3840,26 @@
       }
     ]
   },
+  {
+    "appName": "PlaydioCast: Podcast Radio",
+    "appType": [
+      "app",
+      "podcast player"
+    ],
+    "appUrl": "https://apps.apple.com/app/apple-podcasts/id858974970",
+    "appIconUrl": "playdioCast.png",
+    "platforms": [
+      "iOS",
+      "Mobile",
+      "macOS",
```

Comment:
Can you remove the trailing `,` on this line?

Reply:
done, sorry for that